challenges/Front-End Challenge/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import './App.css';
+import { Navbar, Footer, Form } from './components/index';
+import { loadReCaptcha } from 'react-recaptcha-v3';
+
+function App() {
+
+  useEffect(() => {
```

Comment:
I would like to point out that this is not standard with accessing API keys. Normally you would make a secure request to a protected server to retrieve this key without giving away the value.